packages/block-editor/src/components/link-control/test/index.js

```diff
@@ -227,6 +227,43 @@ describe( 'Basic rendering', () => {
 			expect( isEditing() ).toBe( false );
 		} );
 	} );
+
+	describe( 'Unlinking', () => {
+		it( 'should not show "Unlink" button if no onRemove handler is provided', () => {
+			act( () => {
+				render(
+					<LinkControl value={ { url: 'https://example.com' } } />,
+					container
+				);
+			} );
+
+			const unLinkButton = queryByText( container, 'Unlink' );
```

Comment:
Better to use `getByRole` here to test

1. It's a button.
2. It has the Unlink `name`.
3. It exists.

all at the same time. We might want to still add a `expect().toBeInTheDocument()` though for explicitness.

Reply:
Oh nice!

Reply:
That's [super useful knowledge](https://testing-library.com/docs/queries/byrole/). Thanks @kevin940726. I've updated the tests.